lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/configuration/MainProcessConfig.scala

```diff
@@ -11,13 +11,13 @@ import scala.concurrent.duration.FiniteDuration
   * @param logLevel the logging level
   * @param profilingPath the path to the profiling out file
   * @param profilingTime the time limiting the profiling duration
-  * @param jvmMode if true, enables JVM mode
+  * @param jvm enable JVM mode with default or provided JVM
   * @param extraEnv extra environment variables
   */
 case class MainProcessConfig(
   logLevel: Level,
   profilingPath: Option[Path],
   profilingTime: Option[FiniteDuration],
-  jvmMode: Boolean,
+  jvm: Option[Option[Path]],
```

Comment:
- `jvm` option may be missing or present
- when present it may have no or some JDK path
- hence `Option[Option[Path]]`
- if there is a better way to express this tri-state, I can switch to it...

Reply:
What about having both `jvmMode: Boolean` and `jvmPath: Option[Path]`?

Reply:
I, on the other hand, find the `Option[Option[Path]]` rather easy to follow. Having two values to track would make things harder.

Reply:
- the parameter has to be propagated thru a lot of structures
- splitting it into two arguments would be tedious
- let's keep the current state